app/src/test/java/dev/blaauwendraad/recipe_book/service/UserAuthenticationTest.java

```diff
@@ -64,7 +64,7 @@ void cannotRegisterUserWithExistingUsername() {
     }
 
     @Test
-    void login() throws UserLoginException {
+    void login() throws UserAuthenticationException {
```

Comment:
Can you also add new tests for updating email and password features? I think that would be really good practice